src/Ledger.Net/Requests/Concrete/Public Key Request/PublicKeyHelpers.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.IO;
+
+namespace Ledger.Net.Requests.Helpers
+{
+    public static class PublicKeyHelpers
+    {
+        public static byte[] GetDerivationPathData(
+            uint coinNumber,
+            uint account,
+            uint index,
+            bool isChange,
+            bool isSegwit)
+        {
+            return GetByteData(GetDerivationIndices(coinNumber, account, index, isChange, isSegwit));
+        }
+
+        private static uint[] GetDerivationIndices(
```

Comment:
This wasn't quite finished in my version. It can't hinge on the CoinNumber being 60 because of EthereumClassic, and all the other coin types. I haven't quite decided how this should work, but at the very least we'd need to maintain a list of Bitcoin coins, and Ethereum coins. However, that would make the AddressType enum redundant. The temptation is to hard code coins in to the library, but the problem with that is that new coins are getting minted every day - especially Bitcoin clones. So, the data about the coins needs to come from outside the library.

For now, I'd just say we keep a list of Ethereum coins and Ethereum coins are the exception. Or, pass the AddressType enum in.